pkg/controller/command/didclient/command.go

```diff
@@ -117,3 +123,41 @@ func (c *Command) CreateDID(rw io.Writer, req io.Reader) command.Error {
 
 	return nil
 }
+
+func (c *Command) SaveDIDToSDS(rw io.Writer, req io.Reader) command.Error {
+	didDataToStore := sdscomm.DIDData{}
+
+	err := json.NewDecoder(req).Decode(&didDataToStore)
+	if err != nil {
+		logutil.LogInfo(logger, commandName, saveDIDToSDSCommandMethod, "Failed to decode DID data: "+err.Error())
```

Comment:
I believe there should be LogInfo`f`() function, replace this line with:
`logutil.LogInfof(logger, commandName, saveDIDToSDSCommandMethod, "Failed to decode DID data: %s" err)`

Reply:
There isn't a `LogInfof` function - `logutil` is a utility class inside edge-agent that's used for formatting messages. It uses the formatting directives inside it: https://github.com/trustbloc/edge-agent/blob/master/pkg/controller/command/internal/logutil/util.go

Reply:
hmm, weird, one would think it must exist.

Reply:
The `logutil.LogInfo` function actually calls the underlying logger `Infof` function, so I think that's why